src/js/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Box } from '../Box';
+
+const Avatar = ({
+  align = 'center',
+  round = 'full',
+  size = 'xxsmall',
+  justify = 'center',
+  src,
+  height, // for warning check and discarding the value
```

Comment:
not sure i see the reason for this, i mean if they aren't documented why would people use this?

Reply:
They will be documented on the site as part of referring to all the Box props.

Reply:
ah yeah ok. wondering whether we shouldn't care though, what difference does it make if users set non squared avatars or whatever?

Reply:
I don't want the user to have unexpected side-effects they can't understand, so I rather call it out.